frontend/encore/bootstrap.rst

```diff
@@ -21,55 +27,31 @@ a ``global.scss`` file, import it from there:
     // assets/css/global.scss
 
     // customize some Bootstrap variables
-    $brand-primary: darken(#428bca, 20%);
+    $primary: darken(#428bca, 20%);
 
     // the ~ allows you to reference things in node_modules
-    @import '~bootstrap-sass/assets/stylesheets/bootstrap';
+    @import "~bootstrap/scss/bootstrap";
 
-That's it! This imports the ``node_modules/bootstrap-sass/assets/stylesheets/_bootstrap.scss``
+That's it! This imports the ``node_modules/bootstrap/scss/bootstrap.scss``
 file into ``global.scss``. You can even customize the Bootstrap variables first!
 
 .. tip::
 
     If you don't need *all* of Bootstrap's features, you can include specific files
-    in the ``bootstrap`` directory instead - e.g. ``~bootstrap-sass/assets/stylesheets/bootstrap/alerts``.
-
-After including ``bootstrap-sass``, your Webpack builds might become slow. To fix
```

Comment:
Just asking, why did you remove this explanation about `resolveUrlLoader`? Is it no longer needed in Bootstrap 4? Thanks.

Reply:
I delete this because now on the scss file from bootstrap, we don't need to change the path from the webfont etc, so WebPack work realy fine now... i'm doing some test but for me isn't necessary anymore.